src/BenchmarkDotNet/Toolchains/Mono/MonoAotToolchain.cs

```diff
@@ -18,39 +19,36 @@ public MonoAotToolchain() : base("MonoAot", new Generator(), new MonoAotBuilder(
         {
         }
 
-        public override bool IsSupported(BenchmarkCase benchmarkCase, ILogger logger, IResolver resolver)
+        public override IEnumerable<ValidationError> Validate(BenchmarkCase benchmarkCase, IResolver resolver)
         {
-            if (!base.IsSupported(benchmarkCase, logger, resolver))
+            foreach (var validationError in base.Validate(benchmarkCase, resolver))
             {
-                return false;
+                yield return validationError;
             }
 
             if (!benchmarkCase.Job.Environment.HasValue(EnvironmentMode.RuntimeCharacteristic) || !(benchmarkCase.Job.Environment.Runtime is MonoRuntime))
             {
-                logger.WriteLineError("The MonoAOT toolchain requires the Runtime property to be configured explicitly to an instance of MonoRuntime class");
-                return false;
+                yield return
+                    new ValidationError(true, "The MonoAOT toolchain requires the Runtime property to be configured explicitly to an instance of MonoRuntime class", benchmarkCase);
             }
 
             if ((benchmarkCase.Job.Environment.Runtime is MonoRuntime monoRuntime) && !string.IsNullOrEmpty(monoRuntime.MonoBclPath) && !Directory.Exists(monoRuntime.MonoBclPath))
             {
-                logger.WriteLineError($"The MonoBclPath provided for MonoAOT toolchain: {monoRuntime.MonoBclPath} does NOT exist.");
-                return false;
+                yield return
+                    new ValidationError(true, $"The MonoBclPath provided for MonoAOT toolchain: {monoRuntime.MonoBclPath} does NOT exist.", benchmarkCase);
```

Comment:
```suggestion
                yield return new ValidationError(true, $"The MonoBclPath provided for MonoAOT toolchain: {monoRuntime.MonoBclPath} does NOT exist.", benchmarkCase);
```

Reply:
It was actually me, I do prefer this way, to avoid such a long line.
My IDE does not complain about it as well...

Reply:
It not critical, but the project uses 2 types of formatting for validation errors:
```C#
//#1
yield return new ValidationError(true, "The MonoAOT toolchain requires the Runtime property to be configured explicitly to an instance of MonoRuntime class", benchmarkCase);
//#2
yield return new ValidationError(true,
    "The MonoAOT toolchain requires the Runtime property to be configured explicitly to an instance of MonoRuntime class", 
    benchmarkCase);
```

The perfectionist is crying 😊
![image](https://user-images.githubusercontent.com/19392641/196255616-cbb34d12-6858-4e84-a3ff-4e2b482c5b50.png)

Reply:
> Than this: ![image](https://user-images.githubusercontent.com/40024962/196255815-0b10b40e-601e-4843-98b1-da532e717532.png)

@emanuel-v-r FYI:
Visual Studio split it by 2 lines due to [soft wrap mode](https://learn.microsoft.com/en-gb/visualstudio/ide/reference/how-to-manage-word-wrap-in-the-editor?view=vs-2022
). You can toggle it